### PR TITLE
Worked on AboutUs webpage

### PR DESCRIPTION
**Pull request by joshimehul**
## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [ ] The PR is made from a branch that's **not** called "master".
- [ ] The PR is made from a branch that is up-to-date with "master".
- [ ] The PR's branch is based on "master" and not on any other branch.

### CODE REVIEW

**Comment by jkugsiya** [comment]
@joshimehul  Your page looks good on the desktop version but is not responsive. Please work on making it responsive, maybe use bootstrap for the purpose so that it looks good on small screen sizes too.

**Comment by jkugsiya** [comment]
Also in the PR explanation use x to checkmarks in between the [ ] without any space in between them.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+	font-size: 18px;
+	line-height: 197.69%;
+	/* or 36px */
+
+	letter-spacing: 0.06em;
+
+	color: #000000;
+}
+
+/*end of Mehul's Css
```
Comment (jkugsiya) [inline style.css:128]: The comment is not ended here.

**Review by jkugsiya — CHANGES_REQUESTED** [review]

**Comment by jkugsiya** [comment]
Do not work on master branch. This time its ok but from next time follow [CONTRIBUTING.md](https://github.com/jkugsiya/TechnoIndiaNJR/blob/master/CONTRIBUTING.md)

File: style.css (hunk 1 above)
Comment (joshimehul) [reply style.css:128]: okay

**Pull request closed without merging**